### ISSUE TITLE
[FEAT] Add an example with "real data"

### ISSUE THREAD

**Issue by jbschiratti** [owkin/PyDESeq2#179]
As a PyDESeq2 users, I'd like to have a example/tutorial with "real data" to illustrate PyDESeq2 capabilities.

@tobias-zehnder and @Valerieducret might be interested in this!

**Comment by BorisMuzellec** [comment]
Update: we could do something similar to what is done in https://github.com/owkin/fedpydeseq2 in the tutorials (run a script to download and process a TCGA dataset from the recount3 database).

**Comment by umarteauowkin** [comment]
i agree ! If possible, we should download simpler data however, (in the fedpydeseq2 example, there are multiple sources, which multiplies the risk of bugs). @BorisMuzellec  Would there be a simpler, real life dataset that we could use ?
